src/components/InventoryGroupDetail/InventoryGroupDetail.js

```diff
@@ -21,9 +25,30 @@ const InventoryGroupDetail = ({ groupId }) => {
   const chrome = useChrome();
   const groupName = data?.results?.[0]?.name;
 
+  const { hasAccess: canView } = usePermissionsWithContext(
+    REQUIRED_PERMISSIONS_TO_READ_GROUP(groupId)
+  );
+
+  const EmptyStateNoAccess = () => (
```

Comment:
Let's refactor this empty state outside of this component. Creating a dedicated folder for such prone-to-be-shared components would be great.

Reply:
done!:)